app/models/concerns/work_version_state_machine.rb

```diff
@@ -38,6 +38,18 @@ module WorkVersionStateMachine
         transition %i[first_draft version_draft pending_approval] => :depositing
       end
 
+      # event occurs when a user requests to use globus for upload but still needs to complete initial globus setup
+      event :globus_setup_pending do
+        transition first_draft: :globus_setup_first_draft
+        transition version_draft: :globus_setup_version_draft
+      end
+
+      # event occurs when a user has completed initial globus setup; go back to regular draft state
+      event :globus_setup_complete do
+        transition globus_setup_first_draft: :first_draft
+        transition globus_setup_version_draft: :version_draft
+      end
```

Comment:
Should these two new states be added to line 78, so the `update_metadata` event can still fire off on works in these states?

Reply:
right, and this is needed because a save as draft will trigger that event, correct?  and this should still be allowed when we are pending globus setup

Reply:
@peetucket I believe so